pkgs/development/libraries/rapidcheck/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, cmake, fetchFromGitHub }:
+
+stdenv.mkDerivation rec{
+  name = "rapidcheck-${version}";
+  version = "git-20180927";
```

Comment:
On package naming [the manual](https://nixos.org/nixpkgs/manual/#sec-package-naming) would rather suggest **rapidcheck-unstable-2018-09-27**